jina/serve/networking.py

```diff
@@ -360,14 +409,19 @@ async def send_requests(
                 raise ValueError(f'Unsupported request type {type(requests[0])}')
 
     class _ConnectionPoolMap:
-        def __init__(self, logger: Optional[JinaLogger], summary):
+        def __init__(
+            self,
+            logger: Optional[JinaLogger],
+            metrics: _NetworkingMetrics,
+            runtime_name: str,
```

Comment:
let's try to have similar signature and follow the order: runtime_name, logger, metrics

Reply:
good idea